google-cloud-storage/src/test/java/com/google/cloud/storage/StorageImplMockitoTest.java

```diff
@@ -475,21 +475,21 @@ public void testGetOptions() {
 
   @Test
   public void testCreateBucket() {
-    doReturn(BUCKET_INFO1.toPb())
+    doReturn(Conversions.apiary().bucketInfo().encode(BUCKET_INFO1))
```

Comment:
Do we want to complete moving to mockito from easymock in main branch?

Reply:
Eventually yeah, though I'm not sure it's an immediate need compared to features for gRPC.